target/dovecot/90-sieve.conf

```diff
@@ -31,14 +31,14 @@ plugin {
   # executed. The order of execution within a directory is determined by the
   # file names, using a normal 8bit per-character comparison. Multiple script
   # file or directory paths can be specified by appending an increasing number.
-  #sieve_before =
+  sieve_before = /usr/lib/dovecot/sieve-global/before.dovecot.sieve
```

Comment:
Does this fail when no file it copied in the start-mailserver.sh?

Reply:
Yes, I was afraid of that as well. I couldn't find anything in the Docs though(It just states, that it's safe to not have any files in a directory when configuring a directory as a target) and just ran the tests with it. Dovecot wasn't complaining(the `after.dovecot.sieve` file doesn't exist during the tests) so I basically forgot about it. 
Do you think we should take the safer route and comment those lines out, when not used?

Reply:
I guess it is safer indeed, if you could change it I can merge the request.

Reply:
Let me check again..